tests/third_party/cupy/fft_tests/test_fft.py

```diff
@@ -26,7 +26,7 @@ class TestFft(unittest.TestCase):
         rtol=1e-4,
         atol=1e-7,
         accept_error=ValueError,
-        type_check=has_support_aspect64(),
+        type_check=False,
```

Comment:
Would it be useful to add python version condition for this flag? 
For example
`False if sys.version_info.minor >= 11 else has_support_aspect64()`

Reply:
No, the behavior doesn't depend on python version. It depends on kind of MKL backend used by numpy.
In general, as an improvement, we can investigate how to fetch MKL backed used by numpy in the test. As an idea it might be done like:
```python
numpy.show_config('dicts')['Build Dependencies']["blas"]["name"]
```
But I'd propose to keep currently this out of scope of that PR.